misc-utils/lsfd-sock-xinfo.c

```diff
@@ -1548,3 +1646,550 @@ static void load_xinfo_from_proc_netlink(ino_t netns_inode)
  out:
 	fclose(netlink_fp);
 }
+
+/*
+ * PACKET
+ */
+struct packet_xinfo {
+	struct sock_xinfo sock;
+	uint16_t type;
+	uint16_t protocol;
+	unsigned int iface;
+};
+
+static const char *packet_decode_protocol(uint16_t proto)
+{
+	switch (proto) {
+	case 0:
+		return NULL;
+#ifdef ETH_P_802_3
```

Comment:
All these definitions have been there for a long time, too.

Reply:
Partially done.